libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -1392,12 +1392,10 @@ bool NavEKF3::setOriginLLH(const Location &loc)
     if (!core) {
         return false;
     }
-    if ((sources.getPosXYSource() == AP_NavEKF_Source::SourceXY::GPS) || common_origin_valid) {
-        // we don't allow setting of the EKF origin if using GPS
-        // or if the EKF origin has already been set.
-        // This is to prevent accidental setting of EKF origin with an
-        // invalid position or height or causing upsets from a shifting origin.
-        GCS_SEND_TEXT(MAV_SEVERITY_WARNING, "EKF3 refusing set origin");
+    if (common_origin_valid) {
```

Comment:
I don't think this will work as NavEKF3_core::setOriginLLH() also checks

Reply:
This is resolved now because PeterB has removed the lines from NavEKF3_core::setOriginLLH() as well now.